rs/tests/consensus/utils/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+use std::time::Duration;
+
+/**
+ * 1. Delete existing tc rules (if present).
+ * 2. Add a root qdisc (queueing discipline) for an htb (hierarchical token bucket).
+ * 3. Add a class with bandwidth limit.
+ * 4. Add a qdisc to introduce latency with jitter.
```

Comment:
I think this was removed
```suggestion
 * 4. Add a qdisc to introduce latency.
```

Reply:
Yes, thanks!